### DISCUSSION TITLE
In-Line Icons are Slightly Off-Grid

### DISCUSSION THREAD

**Discussion by BaughBerry** [FlorianWoelki/obsidian-icon-folder#340] [Q&A] [answered]
I'm using the Tabler icons pack, and in-line icons are ever so slightly off-grid, where the top of the icon is higher than the top of the font. The picture below illustrates this. Any ideas how to fix this?

<img width="272" alt="Screenshot 2024-01-03 at 4 00 01 PM" src="https://github.com/FlorianWoelki/obsidian-iconize/assets/129925078/d509f704-bb12-4b93-bc66-d4113e04dade">

**Comment by FlorianWoelki** [comment, marked as answer]
This is for most of the icons at the moment. Could you please create an issue? Thank you
